recipes/keyrings.alt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "keyrings.alt" %}
```

Comment:
I have created PR for conda-docs that mentions dot as a valid character in package name

https://github.com/conda/conda-docs/pull/380